Update 2-Pagerank-RDD.ipynb

chrisdinant opened the pull request:
I added brackets to the pagerank calculation in the Page Contributions Markdown field. Without brackets it returns negative values.

chrisdinant committed:
Update 2-Pagerank-RDD.ipynb

I added brackets to the pagerank calculation in the Page Contributions Markdown field. Without brackets it returns negative values.

andfanilo commented:
You're right, the written formula is off. 

I'm using [this Wikipedia formula](https://wikimedia.org/api/rest_v1/media/math/render/svg/af3909afbed9793d94f41bf99ec98d75d5f1335a) in the tutorial, let's stick close to it and remove the brackets but change the sign of the product term instead. Could you edit the line again ? Thanks !

chrisdinant committed:
Update 2-Pagerank-RDD.ipynb

Changed the pagerank formula to the Wikipedia formulation. https://wikimedia.org/api/rest_v1/media/math/render/svg/af3909afbed9793d94f41bf99ec98d75d5f1335a
